puppet/manifests/site.pp

```diff
@@ -1,63 +1,52 @@
 node default {
-  include profiles::base
```

Comment:
If we remove the last one here, should we at least create an entry for `virt\d+` to keep it managed?

Reply:
oh absolutely, this was more meant as a test on whether it will fix the failures on all the other tests.

I do wonder tho whether we should feed the tests with a dedicated (semi-empty) `site.pp`

Reply:
okay, I dropped the manifest usage in tests and it works, so I added the base profile back to the default node